blockchain/blockchain.go

```diff
@@ -152,7 +156,6 @@ type BlockChain struct {
 
 	mu      sync.RWMutex // global mutex for locking chain operations
 	chainmu sync.RWMutex // blockchain insertion lock
-	procmu  sync.RWMutex // block processor lock
```

Comment:
I don't understand why there was `procmu` in the first place. Could you explain why there was and it could be deleted?

Reply:
@winnie-byun `procmu` was introduced by https://github.com/ethereum/go-ethereum/pull/1917, but I'm not sure why they use this since it is used when setting and getting validator and processor, however, `NewBlockChain` is the only place where the validator and processor is set. Therefore they deleted `procmu`, though it does not seem to be related to this change.